caom2/caom2/tests/data/SampleComposite-CAOM-2.3.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<caom2:Observation xmlns:caom2="http://www.opencadc.org/caom2/xml/v2.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="caom2:CompositeObservation" caom2:id="00000000-0000-0000-5ca9-015f07dcb325" caom2:lastModified="2017-10-10T19:57:05.222" caom2:maxLastModified="2017-10-10T19:57:07.222" caom2:metaChecksum="md5:7d0e7b22db294a2fa7b78d6ef616885e" caom2:accMetaChecksum="md5:d18962d6eb95428bd4502258c81a4106">
+<caom2:Observation xmlns:caom2="http://www.opencadc.org/caom2/xml/v2.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:type="caom2:CompositeObservation" caom2:id="00000000-0000-0000-5ca9-015f07dcb325" caom2:lastModified="2017-10-10T19:57:05.222" caom2:maxLastModified="2017-10-10T19:57:07.222" caom2:metaChecksum="md5:7d0e7b22db294a2fa7b78d6ef616885e" caom2:accMetaChecksum="md5:52b4dd89194b9caab676746b2ec3c1df">

```

Comment:
This is problematic. Checksums are the same as on the Java side and cannot be changed. Output is otherwise correct. I suspect that the reason for the change is the fact that we store the value of em_energy in a different variable. I think you need to change the implementation to keep storing `em_energy` in its own `_em_energy` variable but the setters should not allow for both attributes to be used at the same time. If the old code uses `em_energy` it cannot use `energy_bands` and the other way around.

Reply:
Changed the implementation so that the checksums are not modified.